firefox/background.entry.js

```diff
@@ -28,13 +28,13 @@ addListener('download', ({ url, filename }) => {
 	chrome.downloads.download({ url, filename });
 });
 
-// Firefox doesn't support openerTabId, and needs cookieStoreId to open in correct container
-addListener('openNewTabs', ({ urls, focusIndex }, { index: currentIndex, cookieStoreId }) => {
+// Firefox needs cookieStoreId to open in correct container
+addListener('openNewTabs', ({ urls, focusIndex }, { id: tabId, cookieStoreId }) => {
 	urls.forEach((url, i) => {
 		chrome.tabs.create({
 			url,
 			active: i === focusIndex,
-			index: ++currentIndex,
```

Comment:
Is the `index` no longer necessary? It's still being used in Chrome's background.entry.js.

Reply:
Yes; ideally we would never use `index`, but:
In Chrome if you don't specify `index` all tabs go to the end, instead of being grouped next to the `openerTabId` tab (like ctrl/middle-clicking would do).
In Firefox if you don't specify `index`, new tabs are grouped next to the `openerTabId` tab, or at the end if you've disabled `insertRelatedAfterCurrent`, which is #4230 (either way, the same as ctrl/middle-clicking).